src/main/java/org/zalando/nakadi/repository/TopicRepository.java

```diff
@@ -30,6 +48,9 @@
     Optional<PartitionStatistics> loadPartitionStatistics(Timeline timeline, String partition)
             throws ServiceUnavailableException;
 
+    List<Optional<PartitionStatistics>> loadPartitionStatistics(Collection<TimelineAndPartition> partitions)
```

Comment:
Discussed offline. The contract is to return the same amount of items and in the same order that was requested, I will add comment there.